code/game/objects/items/reagent_containers/glass/bottle.dm

```diff
@@ -188,6 +188,18 @@
 	icon_state = "bottle-5"
 	list_reagents = list(/datum/reagent/medicine/tricordrazine = 60)
 
+/obj/item/reagent_containers/glass/bottle/paracetamol
+	name = "\improper Paracetamol bottle"
+	desc = "A small bottle. Contains paracetamol - a mild, simple painkiller, good for enduring heavy labor."
+	icon_state = "bottle-1"
+	list_reagents = list(/datum/reagent/medicine/paracetamol = 60)
+
+/obj/item/reagent_containers/glass/bottle/isotonic
```

Comment:
Given that the reagent shows up as sal-glu and this is (in principle) for use in chemistry, do you want to label it that way instead? I'm fine with either.